poplar_recovery_builder.sh

```diff
@@ -100,35 +100,45 @@ function parseargs() {
 	# Make sure a single argument was supplied
 	[ $# -lt 1 ] && usage "no partition specified"
 
+	IMAGE_TYPE=Linux
 	INPUT_FILES="L_LOADER USB_LOADER"
 	INPUT_FILES="${INPUT_FILES}"
```

Comment:
This line has no effect and can be deleted.

Reply:
Ok. Will remove.